whois-api/src/main/java/net/ripe/db/whois/api/nrtm4/NrtmHttpControl.java

```diff
@@ -0,0 +1,45 @@
+package net.ripe.db.whois.api.nrtm4;
+
+import org.eclipse.jetty.http.HttpHeader;
+import org.eclipse.jetty.http.HttpScheme;
+import org.eclipse.jetty.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Objects;
+
+@Component
+public class NrtmHttpControl implements Filter {
+
+    @Override
+    public void doFilter(final ServletRequest request, final ServletResponse response, final FilterChain chain) throws IOException, ServletException {
+        final HttpServletRequest httpRequest = (HttpServletRequest) request;
+        if (isHttps(httpRequest)){
+            chain.doFilter(request, response);
```

Comment:
HttpScheme.HTTPS.is(request.getHeader(HttpHeader.X_FORWARDED_PROTO.asString())

Reply:
Ah I see that "is" is ignoring the case, perfect!

Reply:
Updated!